testData/deviceReporter/acceptanceTests/chrome.json

```diff
@@ -1,5 +1,9 @@
 [
     {
         "id": "com.ilunion.cloud4chrome"
```

Comment:
From my conversation with @kaspermarkus in the [fluid-work channel](https://botbot.me/freenode/fluid-work/2017-08-16/?msg=89914142&page=1), we will not remove org.chrome.cloud4chrome and org.mozilla.cloud4firefox yet.